### PR TITLE
fix: Fix typo in provider documentation

### PR DESCRIPTION
**Pull request by tjabbour**

### CODE REVIEW

**Review by laurapacilio — APPROVED** [review]

**Comment by laurapacilio** [comment]
@tjabbour I approved your changes - thank you! Once you sign the CLA (see above), we can merge.

**Comment by tjabbour** [comment]
@laurapacilio Signed!

**Pull request merged**
